Add .pythonrc.py to disable-common.inc

Senemu opened the pull request:
Python 2 executes ``~/.pythonrc.py`` when importing the [``user`` module](https://docs.python.org/2/library/user.html).

Users might also source it from the interactive startup file of their Python interpreter or directly point the [``PYTHONSTARTUP`` environment variable](https://docs.python.org/using/cmdline.html#envvar-PYTHONSTARTUP) to it.

SkewedZeppelin commented:
This should probably also be `noblacklisted` in geany, gnome-builder, pycharm*, etc.

Senemu commented:
I accidentally also changed the order of one other line with my last commit (by sorting the entire section).

I have deleted my fork, so I cannot make any further changes.

Fred-Barclay commented:
I can add to geany etc. Merged -- thanks!
